cellmlmanip/model.py

```diff
@@ -186,6 +199,9 @@ def add_unit(self, units_elements: dict):
     def add_component(self, component: Component):
         """Adds name to list of <component>s in the <model>
         """
+        if component.name in self.components:
+            raise RuntimeError("%s already exists. Check CellML." % component.name)
```

Comment:
Similarly here; generally Python style says try to avoid the more generic error classes. I think `ValueError` is the best fit from https://docs.python.org/3/library/exceptions.html